### ISSUE TITLE
How to find  Training Set and Test Set

### ISSUE THREAD

**Issue by vivianxhh** [Cryptum169/textCNN-for-Chinese#1]
Dear sir:
     How to find  Training Set and Test Set
your sincerely,
Huihui xiao
